sig-cloud-provider/annual-report-2021.md

```diff
@@ -23,62 +32,59 @@ In future, this will be generated from kubernetes/enhancements kep.yaml files
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.beta
+     - [2436 - Leader Migration for Controller Managers](https://github.com/kubernetes/enhancements/tree/master/keps/sig-cloud-provider/2436-controller-manager-leader-migration) - 1.22
    - Alpha
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.alpha
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.alpha
+     - [2133 - Kubelet Credential Provider](https://git.k8s.io/community/$link/README.md) - 1.20
    - Pre-alpha
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md)
+     - [2699 - KEP for adding webhook hosting capability to the CCM framework](https://github.com/cheftako/enhancements/blob/master/keps/sig-cloud-provider/2699-add-webhook-hosting-to-ccm/README.md)
 
 ## Project health
 
 1. What areas and/or subprojects does your group need the most help with?
    Any areas with 2 or fewer OWNERs? (link to more details)
 
    -
-   -
-   -
 
-2. What metrics/community health stats does your group care about and/or measure?
+2. What metrics/community health stats does your group care about and/or
+   measure?
 
-   -
-   -
-   -
+   - We don't currently measure any statistics regarding community health.
 
-3. Does your [CONTRIBUTING.md] help **new** contributors engage with your group specifically by pointing
-   to activities or programs that provide useful context or allow easy participation?
+3. Does your [CONTRIBUTING.md] help **new** contributors engage with your group
+specifically by pointing to activities or programs that provide useful context
+or allow easy participation?
 
-   -
+   - Adding a new CONTRIBUTING.md in the same PR as this report.
 
-4. If your group has special training, requirements for reviewers/approvers, or processes beyond the general [contributor guide],
-   does your [CONTRIBUTING.md] document those to help **existing** contributors grow throughout the [contributor ladder]?
+4. If your group has special training, requirements for reviewers/approvers, or
+processes beyond the general [contributor guide], does your [CONTRIBUTING.md]
+document those to help **existing** contributors grow throughout the
+[contributor ladder]?
 
-   -
+   - N/A (we don't have special requirements beyond the general [contributor
+     guide].
 
 5. Does the group have contributors from multiple companies/affiliations?
 
-   -
+   - Yes, we have contributors from Amazon, Google, VMWare, and others.
 
-6. Are there ways end users/companies can contribute that they currently are not?
-   If one of those ways is more full time support, what would they work on and why?
+6. Are there ways end users/companies can contribute that they currently are
+not?  If one of those ways is more full time support, what would they work on
+and why?
 
-   -
-   -
+   - Yes, we could use help on the extraction/migration effort, including
```

Comment:
are there issues that document this we can link here so that when folks are reading the summary, they will have something to click and take action?